src/extensions/sonarcloud/tasks/SonarCloudPrepare/v2/task.json

```diff
@@ -52,6 +52,24 @@
         "CLI": "Use standalone scanner"
       }
     },
+    {
+      "name": "msBuildVersion",
+      "type": "string",
+      "label": "MSBuild Version",
+      "defaultValue": "6.2.0.85879",
```

Comment:
I wanted to see what the experience would be like if we placed the default scanner values here. It will make it obvious to users what version they are on.

Otherwise, I'm fine with keeping it empty, and we will use the hardcoded value.

Reply:
Great idea IMO, that's how we should use the default value